projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -135,6 +155,51 @@ export class TableWidgetRendererComponent
       value: modeOption
     }));
 
+    this.selectFilterItems$ = forkJoinSafeEmpty(
```

Comment:
Since this is fetching, we should make sure it gets refreshed when the widget refreshes (it also shouldn't fetch until fetchData is called, I think moving this into that function might solve both).